front/pages/home/solutions/customer-support.tsx

```diff
@@ -62,375 +59,267 @@ const pageSettings: pageSettingsProps = {
-            "2xl:col-span-5 2xl:col-start-2",
-            "text-center lg:text-left"
-          )}
-        >
-          <div className="flex w-full px-8 md:px-20 lg:hidden">
-            {MainVisualImage()}
-          </div>
-          <H1 from={pageSettings.from} to={pageSettings.to}>
-            {pageSettings.title}
-          </H1>
-          <P size="lg" className="text-slate-50">
-            {pageSettings.description}
-          </P>
-          <div className="flex gap-4">
-            <Link href="/home/pricing" shallow={true}>
-              <Button
-                variant="highlight"
-                size="md"
-                label="Get started"
-                icon={RocketIcon}
-              />
-            </Link>
-            <Link href="/home/contact" shallow={true}>
-              <Button
-                variant="outline"
-                size="md"
-                label="Talk to sales"
-                target="_blank"
-              />
-            </Link>
-          </div>
-        </div>
-        <div
-          className={classNames(
-            "col-span-12 mx-auto px-8 py-6 sm:max-w-[100%] md:max-w-[90%]",
-            "lg:col-span-7 lg:col-start-6 lg:h-[100%] lg:max-w-[100%]",
-            "2xl:col-span-6",
-            "hidden lg:block"
-          )}
-        >
-          <div className="flex h-full w-full items-center justify-center pt-36 xl:px-8">
-            {MainVisualImage()}
-          </div>
-        </div>
+// Settings for Hero section
+const supportHeroProps = {
+  uptitle: pageSettings.uptitle,
+  title: pageSettings.title,
+  description: pageSettings.description,
+  fromColor: pageSettings.from,
+  toColor: pageSettings.to,
+  visuals: [
+    {
+      src: "/static/landing/support/support1.png",
+      alt: "Support Visual 1",
+      depth: -30,
+    },
+    {
+      src: "/static/landing/support/support2.png",
+      alt: "Support Visual 2",
+      depth: -10,
+    },
+    {
+      src: "/static/landing/support/support3.png",
+      alt: "Support Visual 3",
+      depth: 20,
+    },
+    {
+      src: "/static/landing/support/support4.png",
+      alt: "Support Visual 4",
+      depth: 50,
+    },
+  ],
+  ctaButtons: {
+    primary: {
+      label: "Get started",
+      href: "/home/pricing",
+      icon: RocketIcon,
+    },
+    secondary: {
+      label: "Talk to sales",
+      href: "/home/contact",
+    },
+  },
+};
 
-        <MetricComponent
-          metrics={[
-            {
-              value: "15x",
-              description: <>15x&nbsp;faster to craft an answer</>,
-            },
-            {
-              value: "8h",
-              description: (
-                <>
-                  Save 8&nbsp;hours per&nbsp;agent per&nbsp;week on&nbsp;average
-                </>
-              ),
-            },
-          ]}
-          from="from-amber-200"
-          to="to-amber-500"
-        />
+// Parameters for the Benefits Section
+const supportBenefits: Benefit[] = [
```

Comment:
I think it's ok to inline all of this directly in the component (like it's done for Metrics) - same for supportHeroProps and supportUseCases